app/adapters/application.js

```diff
@@ -1,3 +0,0 @@
-import LFAdapter from 'ember-localforage-adapter/adapters/localforage';
```

Comment:
Yes this file should be removed.  Any file in the app folder will be available within the containing application. Any file in the addon folder will also be available within the containing application but namespaced by the addon name.